nautobot/extras/views.py

```diff
@@ -2423,35 +2423,22 @@ class ObjectMetadataUIViewSet(
 #
 
 
-class NoteView(generic.ObjectView):
-    queryset = Note.objects.all()
-
-
-class NoteListView(generic.ObjectListView):
-    """
-    List Notes
-    """
-
-    queryset = Note.objects.all()
-    filterset = filters.NoteFilterSet
-    filterset_form = forms.NoteFilterForm
-    table = tables.NoteTable
-    action_buttons = ()
-
-
-class NoteEditView(generic.ObjectEditView):
+class NoteUIViewSet(
+    ObjectChangeLogViewMixin, ObjectDestroyViewMixin, ObjectDetailViewMixin, ObjectEditViewMixin, ObjectListViewMixin
+):
+    filterset_class = filters.NoteFilterSet
+    filterset_form_class = forms.NoteFilterForm
+    form_class = forms.NoteForm
     queryset = Note.objects.all()
-    model_form = forms.NoteForm
+    serializer_class = serializers.NoteSerializer
+    table_class = tables.NoteTable
+    action_buttons = None
 
     def alter_obj(self, obj, request, url_args, url_kwargs):
```

Comment:
For Core, I am curious why this is needed.

Reply:
IIRC this is for the case of creating a new Note, automatically setting its `user` field to the user creating the note instead of letting the user specify that information.